demos/simple/index.html

```diff
@@ -6,7 +6,7 @@
     <link rel="stylesheet" href="../../build/css/live-editor.audio.css"/>
     <link rel="stylesheet" href="../../build/css/live-editor.tooltips.css"/>
     <link rel="stylesheet" href="../../build/css/live-editor.ui.css"/>
-    <link rel="stylesheet" href="../../build/css/live-editor.debugger.css"/>
+    <!--<link rel="stylesheet" href="../../build/css/live-editor.debugger.css"/>-->
```

Comment:
This should be removed.  I added code to conditionally load the JavaScript and CSS for debugger only if the user as added ```?debugger=true```.